frontend/js/chatHandler.js

```diff
@@ -99,13 +99,7 @@ class ChatHandler {
   }
   
   async refreshToken() {
-      const formData = new FormData();
-      formData.append("refresh", "");
-
-      const result = await api.post_multipart("/token/refresh/", formData);
-      if (result === null) {
-          this.router.navigate('/home');
-      }
+      const formData = await api.get("/profile")
```

Comment:
Why should refreshToken() call the profile endpoint?

Implemented logic before:
- Before a web socket connection is established, it is checked whether the access token is still valid
- if the access token is not valid anymore, a new access token is requested by calling the refresh-Endpoint
- if that is successful, the web socket connection is opened
- if not, the user is logged out
--> This flow has nothing to do with the /profile endpoint

After calling the api.post_multipart this worked well and also there was no more issue regarding multiple requests to the refresh endpoint